Description: Upstream changes introduced in version 0.12.0-2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 gtk2hs-buildtools (0.12.0-2) unstable; urgency=low
 .
   * Add BangPatterns Language Paragma (Closes: #628302)
 .
 The person named in the Author field signed this changelog entry.
Author: Joachim Breitner <nomeata@debian.org>
Bug-Debian: http://bugs.debian.org/628302

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- gtk2hs-buildtools-0.12.0.orig/c2hs/c/CLexer.x
+++ gtk2hs-buildtools-0.12.0/c2hs/c/CLexer.x
@@ -231,6 +231,7 @@ L\"($instr|@charesc)*\"			{ token CTokSL


 {
+{-# LANGUAGE BangPatterns #-}

 -- We use the odd looking list of string patterns here rather than normal
 -- string literals since GHC converts the latter into a sequence of string
